web/app/routes/application.ts

```diff
@@ -11,6 +11,7 @@ import window from "ember-window-mock";
 import { REDIRECT_STORAGE_KEY } from "hermes/services/session";
 import Transition from "@ember/routing/transition";
 import MetricsService from "hermes/services/metrics";
+import { HERMES_GITHUB_REPO_URL } from "hermes/utils/hermes-urls";
```

Comment:
Not sure if we wanted to use `HERMES_GITHUB_REPO_URL` anywhere?

Reply:
What did you have in mind?

Reply:
Oh, I wasn't sure if you added it to use it somewhere so I was asking. Otherwise, looks good to me!

Reply:
Oh whoops, thanks for catching that. I added it while experimenting and forgot to remove.